scripts/install-bluez-5

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+pushd /tmp/
+
+BLUEZ_VERSION="5.27"
+
+/bin/rm --force --recursive bluez-5.*
+/usr/bin/wget --quiet 'http://www.kernel.org/pub/linux/bluetooth/bluez-${BLUEZ_VERSION}.tar.xz'
+/bin/tar xf bluez-${BLUEZ_VERSION}.tar.xz
+
+pushd bluez-${BLUEZ_VERSION}
+
+./configure --disable-systemd
+/usr/bin/make
+/usr/bin/sudo /usr/bin/make install
+
+popd
+
+# Should we cleanup?
```

Comment:
@pgroudas See comment above... if the source is in `/tmp`, then I'd cleanup.  If it's in `/usr/local/src`, I'd leave it.